src/cmd/manage_nsfs.js

```diff
@@ -822,6 +839,318 @@ function _validate_access_keys(argv) {
         })) throw_cli_error(ManageCLIError.AccountSecretKeyFlagComplexity);
 
 }
+async function tape_mangement(argv) {
+    const action = argv._[1] || '';
+    await manage_tape_operations(action, argv.interval, argv.free_threshold);
+}
+
+async function manage_tape_operations(action, interval, free_threshold) {
```

Comment:
Can we move all these functions to another source file? it becomes too big.

Reply:
Yeah, I'll do that.